src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2593,6 +2599,13 @@ static bool deleteContextAttributeItem
       *currentLocAttrName = "";
     }
 
+    /* Check aspects related to date expiration.
+     * If the target attr is date expiration, nullifying dateExpiration ACTUAL value is the way
+     * of specifying that date expiration field is no longer used */
+    if (targetAttr->name == DATE_EXPIRES) {
```

Comment:
style guide police:
`{` goes in a separate line

Reply:
Fixed in abdaa28